Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/06-NodeDisabling/09-CreateNodeVariantOfDisabledNode.feature

```diff
@@ -118,7 +118,8 @@ Feature: Variation of hidden nodes
     Then I expect node aggregate identifier "the-great-nodini" and node path "court-magician" to lead to no node
 
     When I am in dimension space point {"language":"mul"}
-    Then I expect node aggregate identifier "the-great-nodini" and node path "court-magician" to lead to node cs-identifier;the-great-nodini;{"language":"mul"}
+    Then I expect node aggregate identifier "the-great-nodini" and node path "court-magician" to lead to no node
+  #cs-identifier;the-great-nodini;{"language":"mul"}
```

Comment:
This behavior has changed!
But I doubt that it has been clearly defined before, see [Slack discussion](https://neos-project.slack.com/archives/C04PYL8H3/p1707389969072909)

Reply:
we discussed that the new behavior is better and I added an explicit test for this in `TagSubtree_WithDimensions.feature`